recipes/thorvg/all/conanfile.py

```diff
@@ -33,7 +33,8 @@ class ThorvgConan(ConanFile):
         "with_bindings": [False, 'capi', 'wasm_beta'],
         "with_tools": [False, 'svg2tvg', 'svg2png', 'lottie2gif', 'all'],
         "with_threads": [True, False],
-        "with_vector": [True, False],
+        "with_vector": [True, False],  # removed in thorvg 0.13.1. Renamed to simd
+        "with_simd": [True, False],  # legacy with_vector
```

Comment:
I would mark the legacy option as deprecated (set the default value to `"deprecated"` and log a warning if it is being set).

Reply:
But this case is different. This case is renamed so it is not a "deprecated" issue at all. I think that putting "deprecated" is a bit confusing as it does not exist anymore.